frontend-react/e2e/fixtures/idletimeout.spec.ts

```diff
@@ -0,0 +1,30 @@
+import process from "node:process";
+
+import { test, expect } from "@playwright/test";
+
+const timeout = parseInt(process.env.VITE_IDLE_TIMEOUT ?? "60000");
+// Add/Sub 500 ms to account for variance
+const timeoutLow = timeout - 500;
+const timeoutHigh = timeout + 500;
+
+test.use({ storageState: "playwright/.auth/admin.json" });
+
+test.skip("Does not trigger early", async ({ page }) => {
```

Comment:
Did you want to skip this?

Reply:
Currently admin E2E testing is disabled (per the "setup" project in playwright settings) so I put the skip there for now as a precaution.